assemblies/assembly-audit-log.adoc

```diff
@@ -0,0 +1,36 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="assembly-audit-log"]
+= Audit logs in {product}
+:context: assembly-audit-log
+
+Audit logs are a chronological set of records documenting the user activities, system events, and data changes that affect your {product} users, administrators, or components. Each audit log contains the following information:
+
+* Event name
+* Event actor. For example, terminal, port, location, IP address, or hostname.
+* Event metadata. For example, date, time, and other miscellaneous information.
+* Event status. For example, `success`, `failure`.
+* Severity levels. For example, `info`, `debug`,  `warn`, `error`.
+
+You can use the information in the audit log to achieve the following goals:
+
+Enhance security::
+Trace activities, including those initiated by robots and software templates, back to their source. Know when software templates are executed, as well as the details of application and component installations, updates, configuration changes, and removals.
+
+Automate compliance::
+Use streamlined processes to view log data for specified points in time for auditing purposes or continuous compliance maintenance.
+
+Debug issues::
+Use access records and activity details to fix issues with software templates or plugins.
+
+You can view, search, filter, and manage the log data in a central log management system, such as Splunk or ElasticSearch. You can filter audit logs from other log types by using the `isAuditLog` field.
+
+[NOTE]
+====
+Audit logs are not forwarded to the internal log store by default because this does not provide secure storage. You are responsible for ensuring that the system to which you forward audit logs is compliant with your organizational and governmental regulations, and is properly secured.
+====
+[role="_additional-resources"]
```

Comment:
```suggestion
[NOTE]
====
Audit logs are not forwarded to the internal log store by default because this does not provide secure storage. You are responsible for ensuring that the system to which you forward audit logs is compliant with your organizational and governmental regulations, and is properly secured.
====

[role="_additional-resources"]
```